CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.6.2 / 2017-05-11
+
+* [BUGFIX] Fix potential memory leak in Kubernetes service discovery
+
+## 1.5.3 / 2017-05-11
```

Comment:
He, I had the opposite thought when moving it up here. GitHub sorts by date rather than version two – but that might just be because it can't know it's semver.